src/pybind/mgr/dashboard/controllers/__init__.py

```diff
@@ -949,3 +950,13 @@ def UpdatePermission(func):  # noqa: N802
     """
     _set_func_permissions(func, Permission.UPDATE)
     return func
+
+
+# Empty request body decorator
+
+def allow_empty_body(func):  # noqa: N802
+    """
+    adsfsdfsdf.
```

Comment:
[adsfsdfsdf](https://www.google.com/search?q=adsfsdfsdf)

Now seriously, this is a good chance to explain the underlying issue: POST/PUT requests sending en empty body while JSON requires...

Reply:
Yes forgot to add it. Will add this description